example/src/App.js

```diff
@@ -33,55 +34,80 @@ export default class App extends Component {
     }
   }
 
+  // getAccountToken calls the authorization server to get an access token
+  getAccountToken = async ( accountId) => {
+    let searchParams = new URLSearchParams({
+      accountId: accountId,
+    })
+
+    let options = {
+      method: 'GET',
+      mode: 'cors',
+    };
+  
+    // example auth server url
+    const url = `http://localhost:5000/token?${searchParams.toString()}`
```

Comment:
I wonder if naming the endpoint /token may be a bit confusing for new developers.